### ISSUE TITLE
can´t retrieve google calendars

### ISSUE THREAD

**Issue by bonny547s** [phw198/OutlookGoogleCalendarSync#2011]
I am still configuring ogcs. Have done this in the past succesfully, but it´s failing now.

The field "connected Account" in the settings tab to enter a google calendar is not editable. However, if I klick on "Retrieve Calendars" the correct google account is shown in the "Connected Account" box. 
Immediately after the error message "Failed to retrieve Google calendars. Please check the output on the Sync tab for more details" pops up. 

The Sync Tab sais: "
Unable to get the list of Google calendars. The following error occurred:
Request had insufficient authentication scopes. [403=Forbidden]"

A second error message that sais "You need to select a Google Calendar first on the Settings tab" keeps popping up repeatedly.

I remember from the first (successful) installation, being prompted for some ogcs confirmation when trying to log in to my google account right after having installed ogcs. I am not prompted for this anymore. 
I have uninstalled and reinstalled ogcs, but no success. 

Attached are the logs. 
[OGcalsync.log](https://github.com/user-attachments/files/17846735/OGcalsync.log)

Thanks for any help!

**Comment by PMFlorida** [comment]
Has this been answered?? I can't get my google calendar to sync even if I have added it. I would be more than happy to donate if I could just get this fixed please!
<img width="458" alt="image" src="https://github.com/user-attachments/assets/0c5e066e-4a84-4c12-9820-79c3e2eab74a">

**Comment by phw198** [comment]
Sure has been! Duplicate of #1937 and fixed in v2.11.1 - so you just need to upgrade to the latest alpha.

**Comment by PMFlorida** [comment]
Well not being a technical wizard, I have no idea how to do that.

**Comment by PMFlorida** [comment]
Ok so I found my way to download the version v2-11-1 alpha but now getting
this error message
[image: image.png]

**Comment by phw198** [comment]
:paperclips: Unfortunately GitHub doesn't support submitting files and screenshots to the ticket when replying by email - please log in to GitHub to upload files.

**Comment by bonny547s** [comment]
Yes, the problem is still there. Even with the latest update (2.11.3) it is not possible to edit the "connected account" field in the Settings - Google window. The field is just not editable. There is the text "not connected" that can't be removed in any way. And the "You need to select a Google Caldendar first on the settings tab" Error message keeps popping up every 5 secs...

Since I had previously already tried to install ogcs, could it be that I need to clean up some registry keys after uninstalling the old version?

**Comment by phw198** [comment]
@bonny547s Try clicking the `Retrieve calendars` button...

**Comment by bonny547s** [comment]
Nothing happens obviously, since no account has been selected. I get the message "You need to select a Google Calendar first on the Settings tab".

**Comment by phw198** [comment]
Not obvious at all - a log file would be helpful. You said next to the "Connected Account" it shows "Not Connected", which means you have not retrieved the calendars nor authorised OGCS to access them yet. 

If it is actually showing an email address now (as it did according to your first opening post on this issue), it's possible your installation of OGCS has a Google access token from before they changed the permissions model - which is what #1937 fixed. In that case, click `Disconnect Account` and then `Retrieve Calendars` again.
